src/decaylanguage/decay/decay.py

```diff
@@ -230,18 +256,23 @@ def from_dict(cls, decay_mode_dict):
         except Exception as e:
             raise RuntimeError("Input not in the expected format!") from e
 
-        return cls(bf=bf, daughters=daughters, **dm)
+        return cls(bf=bf, daughters=daughters, **dm)  # type: ignore
```

Comment:
I recommend including the error code, like `# type: ignore[error-code]`, as it explains why this is being ignored, and doesn't hide other errors too.

Reply:
Sounds good. Have not yet learned about this one :D.

Reply:
The codes are printed in brackets at the end of the line (in a different color):

```
src/decaylanguage/modeling/amplitudechain.py:46: error: Need type annotation for "err"  [var-annotated]
```

Not sure how to get it to ingore modeling yet. It's stubbornly checking it.

Reply:
Hmm. Tomorrow I will try and push forward with the remaining bits and will ping you ... Thanks already for starting to look 👍.